tests/test_closing.py

```diff
@@ -2420,6 +2420,8 @@ def try_pay():
                 assert acc['account_resolved']
                 assert acc['resolved_at_block'] > 0
 
+    # Have l1 send all funds to check that the unilateral close info is correct
+    l1.rpc.withdraw(l1.rpc.newaddr('bech32')['bech32'], 'all', minconf=0)
 
 def test_listfunds_after_their_unilateral(node_factory, bitcoind):
```

Comment:
Some small python checking error

```
checking for fcntl... yes
checking whether strnlen is declared... tests/test_closing.py:2426:1: E302 expected 2 blank lines, found 1
yes
make: *** [Makefile:538: check-python-flake8] Error 1
make: *** Waiting for unfinished jobs....
```

Should be fixed with 

```suggestion
    # Have l1 send all funds to check that the unilateral close info is correct
    l1.rpc.withdraw(l1.rpc.newaddr('bech32')['bech32'], 'all', minconf=0)


def test_listfunds_after_their_unilateral(node_factory, bitcoind):
```